Zend/zend_alloc.c

```diff
@@ -3070,25 +3114,28 @@ static void alloc_globals_ctor(zend_alloc_globals *alloc_globals)
 	tmp = getenv("USE_ZEND_ALLOC");
 	if (tmp && !ZEND_ATOL(tmp)) {
 		bool tracked = (tmp = getenv("USE_TRACKED_ALLOC")) && ZEND_ATOL(tmp);
-		zend_mm_heap *mm_heap = alloc_globals->mm_heap = malloc(sizeof(zend_mm_heap));
+		ZEND_ASSERT(sizeof(zend_mm_heap) <= REAL_PAGE_SIZE);
+		zend_mm_heap *mm_heap = alloc_globals->mm_heap = zend_mm_chunk_alloc_int(REAL_PAGE_SIZE * 2, REAL_PAGE_SIZE);
 		memset(mm_heap, 0, sizeof(zend_mm_heap));
```

Comment:
``` suggestion
		memset(mm_heap, 0, REAL_PAGE_SIZE * 2);
```

Or the line could be removed entirely as `zend_mm_chunk_alloc_int()` returns zeroed memory